### PR TITLE
Platforms - UI: fix application page bug

### PR DESCRIPTION
**Pull request by deetz99**
*Issue:*

- bcgov/entity/issues/24883

*Description of changes:*
- bug when clicking begin application
- application override option fix
- update layout name in error.vue

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-platform-dev--pr-379-q0na7d1w.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-platform-dev--pr-379-q0na7d1w.web.app

File: strr-base-web/app/utils/todoItems.ts [hunk 1]
```diff
@@ -18,9 +18,7 @@ export const getTodoApplication = (
       button: {
         label: t('btn.beginApplication'),
         action: async () => {
-          if (applicationInfo) {
-            await navigateTo(localePath(applicationPath))
-          }
+          await navigateTo(localePath(applicationPath))
```
Comment (deetz99) [inline strr-base-web/app/utils/todoItems.ts:21]: do not need application info to start a new application

File: strr-base-web/app/error.vue [hunk 2]
```diff
@@ -21,7 +21,7 @@ useHead({
 </script>
 
 <template>
-  <NuxtLayout>
+  <NuxtLayout name="default">
```
Comment (deetz99) [inline strr-base-web/app/error.vue:24]: must use name in layout or will pickup other layouts

File: strr-base-web/app/composables/useStrrBasePermit.ts [hunk 3]
```diff
@@ -2,7 +2,7 @@ import type { ApiApplicationBaseResp, ApiRegistrationResp } from '~/interfaces/s
 import { downloadFile } from '~/utils/download-file'
 
 export const useStrrBasePermit = <R extends ApiRegistrationResp, A extends ApiApplicationBaseResp, B>() => {
-  const { t } = useI18n()
+  const t = useNuxtApp().$i18n.t // this was casuing an issue when using the composable in route middleware
```
Comment (deetz99) [inline strr-base-web/app/composables/useStrrBasePermit.ts:5]: useI18n composable causing an error when using the base permit composable inside route middleware

File: strr-platform-web/app/middleware/application-page.ts
Comment (deetz99) [inline strr-platform-web/app/middleware/application-page.ts:1]: organize

**Review by thorwolpert — APPROVED** [review]

**Pull request merged**